schema.md

```diff
@@ -0,0 +1,108 @@
+
+Top level keys:
+
+```
+dataset: {}, // Contains main datasets
```

Comment:
Should this be datasets? 
```suggestion
datasets: {}, // Contains main datasets
```

Reply:
https://github.com/covidcanidoit/covidcanidoit/pull/315/files#diff-68cfdabf9fa2ce99fa19fdbe25785f2622056c22f88b1f846d3ac7ffffea0744R22

I'm wondering because of this